projects/rocprofiler-register/source/lib/rocprofiler-register/rocprofiler_register.cpp

```diff
@@ -601,6 +769,47 @@ rocprofiler_register_library_api_table(
 
         if(reginfo) (*reginfo)->propagated = true;
     }
+    else if(_import_match->library_idx == ROCP_REG_HSA &&
+            common::get_env("ROCPROFILER_REGISTER_ATTACHMENT_QUEUES_ENABLED", false))
```

Comment:
So there needs to be a cmake flag that sets a ppdef that, when defined and > 0, sets this to true by default. This allows custom builds which always support attachment